packages/skeleton-common/src/classes/tags-input.ts

```diff
@@ -0,0 +1,18 @@
+import { defineSkeletonClasses } from '../internal/define-skeleton-classes' with { type: 'macro' };
+
+export const classesTagsInput = defineSkeletonClasses({
+	root: 'space-y-2 gap-2',
```

Comment:
Note that on a couple components I've see you do `flex flex-col gap-2` on the root element. But unfortunately this has a side effect of making the items within stretch to fill the width. This is why the "clear files" button was so wide.

By switching to `w-full space-y-2` we get the same look, without the side effect.

I've already applied this to the FileUpload - but keep in mind for other components in the future.